js/envirophat/README.md

```diff
@@ -0,0 +1,69 @@
+# Parts list:
+* Raspberry Pi Zero W  (https://www.adafruit.com/product/3410) $34.5 
+* You will USB powered hub, USB keyboard, USB mouse, HDMI cable and a Monitor with HDMI
```

Comment:
You will *need a* USB powered hub, *a* USB keyboard, *a* USB mouse, *a* HDMI cable and a Monitor with HDMI